www/docs/api/index.php

```diff
@@ -197,7 +197,7 @@ function with the <em>callback</em> variable, and then that function will be
 <p>Low volume, charitable use of the API service itself is free. This means
 direct use by registered charities, or individuals pursuing a non-profit
 project on an unpaid basis, with a volume of up to 50,000 calls per year.
-Please email us at <a href="mailto:<?=$em?>"><?=$em?></a> for all other use. If you are using
+Please see our <a href="pricing">pricing page</a> for all other use. If you are using
```

Comment:
This should either be `pricing.php`, or there should be a `pricing` subfolder containing an `index.php`.